configure.ac

```diff
@@ -194,7 +194,7 @@ AC_ARG_ENABLE(wayland,
 # Check if we have gtk-layer-shell installed, and thus should build with Wayland support
 have_wayland=no
 if test "x$enable_wayland" != "xno"; then
-  PKG_CHECK_MODULES(GTK_LAYER_SHELL, gtk-layer-shell-0, have_wayland=yes, [
+  PKG_CHECK_MODULES(WAYLAND, gtk-layer-shell-0 wayland-client, have_wayland=yes, [
         if test "x$enable_wayland" = "xyes"; then
```

Comment:
So, mate-panel depends on libwayland and gtk-layer-shell now?

Reply:
Only if Wayland is to be enabled:
`if test "x$enable_wayland" != "xno"; then`

Reply:
The question wasn't if wayland support is enabled or not. I always build with wayland and X11 support since wayland is supported.
But Ok, i have to add wayland-devel as build requires

Reply:
Correct. Previously all of the Wayland interaction was done via GTK and GTK Layer Shell, but to support the foreign toplevel protocol the applet needs to use libwayland directly.